content/en/docs/concepts/traffic-management/index.md

```diff
@@ -1050,62 +637,37 @@ spec:
-    request. A retry is an attempt to complete an operation multiple times if
-    it fails. You can set defaults and specify request-level overrides for both
-    timeouts and retries or for one or the other.
+## Network resilience and testing {#network-resilience-and-testing}
 
-- **Circuit breakers**
+As well as helping you direct traffic around your mesh, Istio provides opt-in
+failure recovery and fault injection features that you can configure dynamically
+at runtime. Using these features helps your applications operate reliably,
+ensuring that the service mesh can tolerate failing nodes and preventing
+localized failures from cascading to other nodes.
 
-    Circuit breakers prevent your application from stalling as it waits for an
-    upstream service to respond. You can configure a circuit breaker based on a
-    number of conditions, such as connection and request limits.
+### Timeouts {#timeouts}
 
-- **Fault injection**
+A timeout is the amount of time that an Envoy proxy should wait for replies from
+a given service, ensuring that services don’t hang around waiting for replies
+indefinitely and that calls succeed or fail within a predictable timeframe. The
+default timeout for HTTP requests is 15 seconds, which means that if the service
+doesn’t respond within 15 seconds, the call fails.
 
-    Fault injection is a testing method that introduces errors into a system to
-    ensure that it can withstand and recover from error conditions. You can
-    inject faults at the application layer, rather than the network layer, to
-    get more relevant results.
-
-- **Fault tolerance**
-
-    You can use Istio failure recovery features to complement application-level
-    fault tolerance libraries in situations where their behaviors don’t
-    conflict.
-
-{{< warning >}}
-While Istio failure recovery features improve the reliability and availability
-of services in the mesh, applications must handle the failure or errors and
-take appropriate fallback actions. For example, when all instances in a load
-balancing pool have failed, Envoy returns an `HTTP 503` code. The application
-must implement any fallback logic needed to handle the `HTTP 503` error code
-from an upstream service.
-{{< /warning >}}
-
-## Timeouts and retries
-
-You can use Istio's traffic management resources to set defaults for timeouts
-and retries per service and subset that apply to all callers.
-
-### Override default timeout setting
-
-The default timeout for HTTP requests is 15 seconds. You can configure a
-virtual service with a routing rule to override the default, for example:
+For some applications and services, Istio’s default timeout might not be
+appropriate. For example, a timeout that is too long could result in excessive
+latency from waiting for replies from failing services, while a timeout that is
+too short could result in calls failing unnecessarily while waiting for an
+operation involving multiple services to return. To find your optimal timeout
```

Comment:
"To find your optimal ..."? Do you mean "To tune your optimal ..."? Or maybe "To find and use your optimal ..."

Reply:
The latter!